DEPLOYMENT.md

```diff
@@ -0,0 +1,62 @@
+# Configuring RHMAP Core to use non-wildcard SSL routing
+
+For a standard RHMAP installation deployed on top of OpenShift, we recommend that a wild card SSL certificate be installed on the OpenShift router. However, in situations where this is not desirable, RHMAP can be exposed using single URL.
+
+## RHMAP Proxy
+
+RHMAP Proxy is a separate component which should be deployed to OpenShift per MBaaS project. Its purpose is to route requests from a single external host to cloud applications and RHMAP Core instances deployed on OpenShift.
+
+
+For example, when RHMAP Proxy receives a request for `https://rhmapproxy.internal.com/cloud_app_id/hello`, it will be translated and reverse proxied to `http://cloud_app_id.internal.com/hello`.
+
+The RHMAP Proxy application is bundled within the RHMAP RPM in the form of an OpenShift template. Creating this application will create an OpenShift route which will expose this service. Carry out the following steps to configure RHMAP Core to use non-wildcard routing.
+
+**_1.  Setting Millicore environment variable for client side connectivity_**
```

Comment:
`Setting Millicore environment variable for client side connectivity`
This suggest that environment variable is named `Millicore` 
Let's change this to actual name of the env variable.